scripts/write_complete_language_data.py

```diff
@@ -140,22 +146,29 @@ def write_complete_data():
                 continue
 
             for keys, vals in requisite_data.items():
-                if _is_valid(keys):
+                if _is_valid_cardinal(keys):
                     for key, val in vals.items():
                         # Removing soft-hyphens from the source file.
                         val = val.replace('\xad', '')
-                        _extract_information(key, val, language_data)
+                        _extract_information(key, val, language_data["NUMBERS"])
 
         with open(full_supplementary_path, 'r') as supplementary_data:
             data = json.load(supplementary_data)
-            for keys in REQUIRED_NUMBERS_DATA:
-                language_data[keys].update(data[keys])
-                sorted_tuples = sorted(language_data[keys].items(), key=lambda x: (x[1], x[0]))
-                for items in sorted_tuples:
-                    word, number = items[0], items[1]
-                    ordered_language_data[keys][word] = int(number)
+            for number_types in TYPES_OF_NUMBERS:
+                for keys in REQUIRED_NUMBERS_DATA:
+                    language_data[number_types][keys].update(data[number_types][keys])
+                    sorted_tuples = sorted(language_data[number_types][keys].items(), key=lambda x: (x[1], x[0]))
+                    for items in sorted_tuples:
+                        word, number = items[0], items[1]
+                        ordered_language_data[number_types][keys][word] = int(number)
             skip_tokens = sorted(data["SKIP_TOKENS"])
+            is_long = data["IS_LONG"]
         ordered_language_data["SKIP_TOKENS"] = skip_tokens
+        if is_long:
```

Comment:
So initially i was just writing boolen true and false into the dict. However there was an issue while writing to python files using json dumps. 

json has the boolean as 'false' which was being written into python files but python has boolean as 'False'. I couldn't see an elegant way so used 1/0 instead of True/False. 
https://github.com/morse-simulator/morse/issues/562 might help to expand what exactly the issue is.